libraries/ESP8266WiFi/library.properties

```diff
@@ -7,3 +7,4 @@ paragraph=With this library you can instantiate Servers, Clients and send/receiv
 category=Communication
 url=
 architectures=esp8266
+dot_a_linkage=true
```

Comment:
I don't understand why these are all being converted to .a.  Does this mean anyone providing a library for the 8266 now needs to modify their lib.properties?

Reply:
This is not mandatory (except for the ESP8266WiFi). It is not needed by anyone. Such library's object files are stored in an intermediate libraryname.a (the regular way after all) and this one is used instead of all library's object files in the linker command line.
Other libraries not using "dot_a_linkage" are still usable.
I checked with ESP8266Audio (local branch with this PR merged with latest master) and it's OK. I also tried with "dot_a_linkage" in your lib and the final sketch was reduced by 25KB (!).
Not to mention the linker command looking like a real one (with some .a instead of a huge bunch of .o - well, this is minor cosmetic effect, but still :) ).
What I do not understand is why this was not done by default from the beginning in Arduino.

Reply:
That's awesome news!  Glad to hear about it, I'll throw it in my own lib as well now...